Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -309,3 +309,26 @@ export function transformPlane(planeToTransform, targetPlane, viewType) {
   planeToTransform.setPoint1(rotatedPoint1);
   planeToTransform.setPoint2(rotatedPoint2);
 }
+
+/**
+ * Get the orthogonal plane name of 'planeName' in a specific 'rotatedPlaneName'
+ * Example: planeName='X' on rotatedPlaneName='Z', then the associated plane name
+ * of 'X' plane is 'Y'
+ * @param {String} planeName
+ * @param {String} rotatedPlaneName
+ */
+export function getAssociatedPlaneName(planeName, rotatedPlaneName) {
```

Comment:
`getAssociatedLinesName` and `getAssociatedPlaneName` do the same. Plane association should be computed in only one function (no code duplication !).

It is therefore a matter of conversion. e.g. if you had `getLineNameFromPlaneName(planeName, rotatedPlaneName)` and `getPlaneNameFromLineName(lineName)`, then:
```
export function getAssociatedPlaneName(planeName, rotatedPlaneName) {
  const lineName = getLineNameFromPlaneName(planeName, rotatedPlaneName);
  const associatedLineName = getAssociatedLinesName(lineName);
  return  getPlaneNameFromLineName(associatedLineName);
```

You can also decide to do the opposite (it might actually make more sense): keep `getAssociatedPlaneName` but make `getAssociatedLinesName` call it.

Reply:
It is not clear to me when you should use a plane name (X, Y, Z) and when you use a view type (YZ_PLANE, XZ_PLANE, XY_PLANE). It would be great to have a rule and stick to it.

Reply:
Is it `getAssociatedPlaneName`  or `getAssociatedViewPlaneName` ? (see `getViewPlaneNameFromViewType`).

I do not have a preference, but we should be consistent.